packages/server/src/db/index.js

```diff
@@ -340,6 +340,16 @@ async function getGrant({ grantId }) {
     return results[0];
 }
 
+async function getClosestGrants() {
+    const timestamp = new Date();
+    const query = await knex(TABLES.grants)
+        .select('title', 'close_date')
+        .where('close_date', '>=', timestamp)
+        .orderBy('close_date', 'asc')
+        .limit(3)
```

Comment:
future iterations might take the limit as in API input from the client instead of "hard-coding" 3